src/notifications/using_webhooks.md

```diff
@@ -0,0 +1,99 @@
+# Using Webhooks
+
+## Registering a Webhook
+
+Once your vendor [NDS is set up](setup_nds.md) and can accept POST requests from the SDK services, you can within your main application register the webhook URL with the SDK by calling the register webhook API as follows:
```

Comment:
I think the snippets make it clear that the register webhook API is part of SDK.
```suggestion
Once your vendor [NDS is set up](setup_nds.md) and can accept POST requests from the SDK services, you can register the webhook URL within your main application by calling the register webhook API as follows
```